ajax/libs/bulmaswatch/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "bulmaswatch",
+  "description": "Themes for Bulma",
+  "author": "Jenil Gogari <i@jgog.in>",
+  "homepage": "https://jenil.github.io/bulmaswatch",
+  "filename": "bulmaswatch.min.css",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/jenil/bulmaswatch.git"
+  },
+  "license": "MIT",
+  "keywords": [
+    "css",
+    "sass",
+    "flexbox",
+    "responsive",
+    "framework",
+    "themes"
+  ],
+  "autoupdate": 
+  {
+    "source": "git",
+    "target": "git://github.com/jenil/bulmaswatch.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "bulmaswatch*.+(js|css|map)"
```

Comment:
@hare1039 I think we can let author @jenil decide this.
But at first time, the author write the default/ as the main file, so I think just need `default`.
ref: https://github.com/cdnjs/cdnjs/pull/11322/commits/6b1306274100a630b33854c7ffb843f432bdfde2